client/src/components/organizations/OrganizationPkiRequest.vue

```diff
@@ -44,12 +45,23 @@
 import { JoinRequestStatus, LocalJoinRequest } from '@/parsec';
 import { IonCardContent, IonIcon, IonText, IonButton } from '@ionic/vue';
 import { arrowForward, closeCircle } from 'ionicons/icons';
-import { computed } from 'vue';
+import { attachMouseOverTooltip } from 'megashark-lib';
+import { computed, onMounted, useTemplateRef } from 'vue';
+
+const statusTextRef = useTemplateRef<InstanceType<typeof IonText>>('statusText');
 
 const props = defineProps<{
   request: LocalJoinRequest;
 }>();
 
+onMounted(async () => {
+  if (props.request.status === JoinRequestStatus.Pending) {
```

Comment:
Two problems:
- request can change without the component being unmounted/mounted, which means it will display the right text but the wrong tooltip. The prop needs to be watched.
- If I see another `?` that's just here to shut the linter up, I'll curse you so that you'll never be able to eat tomato sauce without some splashing on your shirt.